### ISSUE TITLE
Feature RQ: CSV Export for Shure WWB

### ISSUE THREAD

**Issue by zacpaton** [AlexandreRouma/SDRPlusPlus#1560]
**Feature description**
I would love if this app would support exporting to CSV with the following format, as this could enable a lot of live audio engineers to use this amazing scan app to import to Shure Wireless Workbench and then do frequency coordination using the data with a cheaper scanner than the much more expensive alternatives.

https://content-files.shure.com/Pubs/WWB/en-US/en-US/scanning.html
![image](https://github.com/user-attachments/assets/9388531a-3d62-4619-976d-a69b9fc785c9)

**Comment by AlexandreRouma** [comment]
No, the frequency manager will only ever support its own format for import export.
